### PR TITLE
Partner form

### PR DESCRIPTION
**Pull request by mbouzi**
- partner button links at "partner" and "/tech-partner" now go to "/partner-form"
- error messages for required inputs
- user email sent to customer.io
- user object with all data points sent to postgres db

### CODE REVIEW

File: layouts/partner-form/single.html [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+      analytics.identify(analytics.user().anonymousId(),
+        {email: user.email, timestamp: user.timestamp, partner_form: true},
+        {integrations: {'All': false, 'Customer.io': true}}, function() {
+          analytics.track("partner-form", {});
+      });
+
+      analytics.identify(analytics.user().anonymousId(),
+        user,
+        {integrations: {'All': false, 'postgresSelfHosted': true}}, function() {
+          window.location = "/thank-you"
```
Comment (pgerbes1) [inline layouts/partner-form/single.html]: On mobile I get a weird display once done filling out the form
image

File: layouts/partner-form/single.html [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+           }
+        }
+      }
+
+      // timestamps with timezone
+      user.timestamp = Math.floor((new Date()).getTime() / 1000)
+
+      analytics.identify(analytics.user().anonymousId(),
+        {email: user.email, timestamp: user.timestamp, partner_form: true},
+        {integrations: {'All': false, 'Customer.io': true}}, function() {
```
[the remark this answers is not in the document]
Comment (mbouzi) [reply layouts/partner-form/single.html:165]: @pgerbes1 I haven't set that up in my code, maybe you've signup up for something separate with that same email?

File: layouts/partner-form/single.html [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+          analytics.track("partner-form", {});
+      });
+
+      analytics.identify(analytics.user().anonymousId(),
+        user,
+        {integrations: {'All': false, 'postgresSelfHosted': true}}, function() {
```
[the remark this answers is not in the document]
Comment (mbouzi) [reply layouts/partner-form/single.html:171]: @pgerbes1 think I was able to make the change, should be double checked

**Comment by grafael** [comment]
It's not possible to "unselect" cases from step **8** :

select_tardigrade

**Comment by mbouzi** [comment]
> It's not possible to "unselect" cases from step **8** :
> 
> select_tardigrade

this element was harder than I thought it'd be to create deselect functionality for, but I was able to do it on double-clicks

With just onclick, "event.preventDefault" does not work for "select" elements so it was selecting and deselecting within one click

**Pull request merged**